src/gamecontroller.cs

```diff
@@ -535,4 +621,1162 @@ interface GCEventViewController {
+
+	[TV (14,0), Mac (11,0), iOS (14,0)]
+	[BaseType (typeof (GCPhysicalInputProfile))]
+	interface GCKeyboardInput
+	{
+		[NullAllowed, Export ("keyChangedHandler", ArgumentSemantic.Copy)]
+		GCKeyboardValueChangedHandler KeyChangedHandler { get; set; }
+
+		[Export ("anyKeyPressed")]
+		bool AnyKeyPressed { [Bind ("isAnyKeyPressed")] get; }
+
+		[Export ("buttonForKeyCode:")]
+		[return: NullAllowed]
+		GCControllerButtonInput Button (nint code);
+	}
+
+	[Mac (11, 0), iOS (14, 0), TV (14, 0)]
+	[BaseType (typeof (NSObject))]
+	interface GCMouse : GCDevice
+	{
+		[NullAllowed, Export ("mouseInput", ArgumentSemantic.Strong)]
+		GCMouseInput MouseInput { get; }
+
+		[Static]
+		[NullAllowed, Export ("current", ArgumentSemantic.Strong)]
+		GCMouse Current { get; }
+
+		[Static]
+		[Export ("mice")]
+		GCMouse[] Mice { get; }
+
+		[Notification, Field ("GCMouseDidConnectNotification")]
+		NSString DidConnectNotification { get; }
+
+		[Notification, Field ("GCMouseDidDisconnectNotification")]
+		NSString DidDisconnectNotification { get; }
+
+		[Notification, Field ("GCMouseDidBecomeCurrentNotification")]
+		NSString DidBecomeCurrentNotification { get; }
+
+		[Notification, Field ("GCMouseDidStopBeingCurrentNotification")]
+		NSString DidStopBeingCurrentNotification { get; }
+	}
+
+	delegate void GCMouseMoved (GCMouseInput mouse, float deltaX, float deltaY);
+
+	[Mac (11, 0), iOS (14, 0), TV(14, 0)]
+	[BaseType (typeof (GCControllerDirectionPad))]
+	interface GCDeviceCursor {}
+
+	[Mac (11,0), iOS (14,0), TV (14,0)]
+	[BaseType (typeof (GCPhysicalInputProfile))]
+	interface GCMouseInput
+	{
+		[NullAllowed, Export ("mouseMovedHandler", ArgumentSemantic.Copy)]
+		GCMouseMoved MouseMovedHandler { get; set; }
+
+		[Export ("scroll")]
+		GCDeviceCursor Scroll { get; }
+
+		[Export ("leftButton")]
+		GCControllerButtonInput LeftButton { get; }
+
+		[NullAllowed, Export ("rightButton")]
+		GCControllerButtonInput RightButton { get; }
+
+		[NullAllowed, Export ("middleButton")]
+		GCControllerButtonInput MiddleButton { get; }
+
+		[NullAllowed, Export ("auxiliaryButtons")]
+		GCControllerButtonInput[] AuxiliaryButtons { get; }
+	}
+
+	interface IGCDevice {}
+
+	[TV (14,0), Mac (11,0), iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[Protocol]
+	[Model]
+	interface GCDevice
+	{
+		[TV (9, 0), Mac (10, 9), iOS (7, 0)]
+		[Abstract]
+		[Export ("handlerQueue", ArgumentSemantic.Strong)]
+		DispatchQueue HandlerQueue { get; set; }
+
+		[TV (9, 0), Mac (10, 9), iOS (7, 0)]
+		[Abstract]
+		[NullAllowed, Export ("vendorName")]
+		string VendorName { get; }
+
+		[TV (13, 0), Mac (10, 15), iOS (13, 0)]
+		[Abstract]
+		[Export ("productCategory")]
+		string ProductCategory { get; }
+
+		[TV (14, 0), Mac (11, 0), iOS (14, 0)]
+		[Abstract]
+		[Export ("physicalInputProfile", ArgumentSemantic.Strong)]
+		GCPhysicalInputProfile PhysicalInputProfile { get; }
```

Comment:
The availability attributes do not make sense here, specially since the container protocol is `[TV (14,0), Mac (11,0), iOS (14,0)]` so someone is lying to us

Reply:
This is bizarre because the availability attributes in the header match these ones...but it's impossible for that to be true. I'm thinking we should just leave the container protocol's availability attribute and remove the rest of them so that everything has `[TV (14,0), Mac (11,0), iOS (14,0)]`.